### DISCUSSION TITLE
apiURL issue in Texture Systems

### DISCUSSION THREAD

**Discussion by GilroyMTA** [OwlGamingCommunity/MTA#6] [Support] [answered]
As in the title, apiURL is looking depreceated. There is no way I can change the Client ID or print the problem to my debugscript. API ERROR 35's source is not reachable.

**Comment by ChaosParadise** [comment, marked as answer]
According to Imgur docs, it is still available. https://apidocs.imgur.com/#2078c7e0-c2b8-4bc8-a646-6e544b087d0f

That error more sounds like your server isn't able to reach out to the internet.

**Reply by GilroyMTA** [reply]
Yeah. The server wasn't able to reach out to the internet. I solved that by moving the server to another one. Thanks in advance for giving a tip!
